defaults/main.yml

```diff
@@ -15,3 +15,13 @@ memcached_cfg_file: '/etc/memcached.conf'
 memcached_consul_tags_default: ['memcached', 'cache']
 memcached_consul_tags_extra: []
 memcached_consul_tags: '{{ memcached_consul_tags_default + memcached_consul_tags_extra }}'
+
+memcached_exporter_enabled: true
+memcached_exporter_port: 9150
+memcached_exporter_version: '0.8.0'
+memcached_exporter_release_name: 'memcached_exporter-{{ memcached_exporter_version }}.linux-amd64'
+memcached_exporter_url: 'https://github.com/prometheus/memcached_exporter/releases/download/v{{ memcached_exporter_version }}/{{ memcached_exporter_release_name }}.tar.gz'
+memcached_exporter_checksum: '0a666f41d733ebdc20cf2dc66fbb0c2899afc237002db67b7a31ff7f9f7dde3f'
+
+memcached_exporter_consul_service: 'memcached_exporter'
+memcached_exporter_consul_tags: '{{ memcached_consul_tags }} + ["memcached_exporter"]'
```

Comment:
Are you sure this works? Shouldn't the additional array be adde within the curly braces?

Also, why put the service name as a tag? Just add the individual words. Like `["exporter", "metrics"]`.

Reply:
It might actually work. I really hate this string interpolation syntax in Ansible. Yaml files are horrible for this kinds of stuff.

Reply:
I've applied it on master-01 and it works (I didn't know YAML can merge lists) but you're right it should be in the braces so ansible/python can do it